main.py

```
@@ -1,11 +1,273 @@
+    def open_reader(self, comic: dict):
+        cont = ReadingController(comic)
+        cont.read_comic()
+        
+    def print_hi(self):
+        print("Hi")
+
+    def create_continue_reading_area(self, list_of_comics_marked_as_read):
+        return self.create_scroll_area(list_of_comics_marked_as_read, header="Continue Reading", upon_clicked=self.open_reader)
+    
+    def create_recommended_reading_area(self, list_of_recommended_comics):
+        return self.create_scroll_area(list_of_recommended_comics, header="Recommended Next Read")
```

Pylint (3.3.7)

[error] No value for argument 'upon_clicked' in method call

(E1120)